index.js

```diff
@@ -178,6 +179,7 @@ async function getIPAddress(serverId) {
     return ipAddress;
   } catch (error) {
     core.setFailed(`Failed to get IP Address: ${error.message}`);
```

Comment:
I think we should change these other `setFailed` calls to plain old logs

Reply:
My learning has been that you don't wnat to use console functions in github actions. Instead, I'll switch them to core.info, which I believe gets all the things you want, without the console writing issues.

Reply:
sorry, i'll set it to core.error